src/lib/controllers/ui-controller.js

```diff
@@ -576,7 +576,8 @@ export default class UIController {
         },
 
         setWordLists (state, wordLists) {
-          state.hasWordListsData = Boolean(wordLists.find(wordList => !wordList.isEmpty))
+          let checkWordLists = Array.isArray(wordLists) ? wordLists : Object.values(wordLists)
```

Comment:
I asked this on #442 but should have asked it here -- why do we get different objects here? maybe it's better to always pass in a consistent object?

Reply:
we have this difference here because it is used in two scenarios:
1) when a wordlist is updated - then we pass it as an array
2) when we initLists - then we pass all wordlist as an object with wordLists for various languages

in both workflows we get if one or any wordlist has items and update wordListUpdateTime
in both scenarios the checking code is the same - we only need to do this check. 
That's why I want to save it as-as.

Reply:
ok